src/cljs/rems/util.cljs

```diff
@@ -88,7 +88,9 @@
    (fn [resolve reject]
      (GET url (-> (merge {:response-format :transit}
                          opts
-                         {:error-handler (wrap-default-error-handler (:error-handler opts))})
+                         (if (:simple-error-handler? opts)
```

Comment:
Could document this option in the docstring. The current docstring is a bit weird, but a short mention about this option would fit I think.

Reply:
did that